ui/src/authors/components/AuthorName.jsx

```diff
@@ -5,15 +5,21 @@ import { getAuthorDisplayName } from '../utils';
 
 class AuthorName extends Component {
   render() {
-    const { name } = this.props;
+    const { name, nativeName } = this.props;
     const displayName = getAuthorDisplayName(name);
 
-    return <span>{displayName}</span>;
+    return (
+      <span>
+        {displayName}
+        {nativeName && <span> ({nativeName})</span>}
+      </span>
+    );
   }
 }
 
 AuthorName.propTypes = {
   name: PropTypes.instanceOf(Map).isRequired,
+  nativeName: PropTypes.string,
```

Comment:
`nativeName` is already inside `name` prop, so this isn't needed.

Reply:
right, missed this